codeship-steps.yml

```diff
@@ -0,0 +1,3 @@
+- name: unittests
```

Comment:
It looks like Codeship will only be running the two new unit tests, not all of the functional (Behat) tests. Would you mind adding the necessarily lines to these codeship files to run the Behat tests as well?

Reply:
The information for the codeship-services.yml file can be copied nearly verbatim from the docker-compose.yml file's entries for running the Behat tests.

Reply:
I'll have another look. I tried it once and there were too many complications that were tripping me up, e.g. you can't specify ports in the codeship files.

Reply:
Looks like I got it passing.  I guess it wasn't the ports that were an issue.